pkgs/games/fteqw/generic.nix

```diff
@@ -0,0 +1,55 @@
+{ pname
+, releaseFile
+, buildFlags
+, buildInputs
+, nativeBuildInputs ? []
+, postFixup ? ""
+, description
+, ... }@args:
+
+with args;
+
```

Comment:
```suggestion
```
please prefix relevant variables instead

Reply:
I've removed it entirely. It turns out to not be needed at all.